hack/metric_name_linter.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  * See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2023 Red Hat, Inc.
+#
+#
+source hack/common.sh
+
+fail_if_cri_bin_missing
+
+set -e
+
+PROJECT_ROOT="$(readlink -e "$(dirname "${BASH_SOURCE[0]}")"/../)"
+metrics_doc_path="${PROJECT_ROOT}/docs/metrics.md"
+linter_image_tag="v0.0.1"
+
+# Parse command-line arguments
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+    --operator-name=*)
+        operator_name="${1#*=}"
+        shift
+        ;;
+    --sub-operator-name=*)
+        sub_operator_name="${1#*=}"
+        shift
+        ;;
+    *)
+        echo "Invalid argument: $1"
+        exit 1
+        ;;
+    esac
+done
+
+# Get the metrics list
+go build -o _out/prom-metrics-collector "${PROJECT_ROOT}"/tools/prom-metrics-collector/...
```

Comment:
@nunnatsa @enp0s3 should /tools/prom-metrics-collector/ also be a parameter? Doesn every operator have a /tools sub folder?

Reply:
@sradco After consulting with @enp0s3 paths changed to env vars.